test/mod/test_testutil_dnf4.py

```diff
@@ -1,10 +1,12 @@
 import os.path
+import platform
 
 import pytest
 
 testutil_dnf4 = pytest.importorskip("osbuild.testutil.dnf4")
 
 
+@pytest.mark.skipif(platform.machine() != "x86_64", reason="Test data contain only x86_64 arched pkgs")
```

Comment:
I wonder if we should solve this the other way around; the dependency solving is pretty important to work so maybe we'd be better off to include packages for the other architectures instead.

@thozza what do you think? Since you've done most of the depsolve testing work so far.